internal/provisioner/registry/registry.go

```diff
@@ -19,7 +20,7 @@ const (
 )
 
 func HandleBundleDeployment(ctx context.Context, fsys fs.FS, bd *rukpakv1alpha2.BundleDeployment) (*chart.Chart, chartutil.Values, error) {
-	plainFS, err := convert.RegistryV1ToPlain(fsys, bd.Spec.WatchNamespaces)
+	plainFS, err := convert.RegistryV1ToPlain(fsys, bd.Spec.InstallNamespace, []string{metav1.NamespaceAll})
```

Comment:
I assume removal of `watchNamespace` field would be a follow up?

Reply:
You mean the `RegistryV1ToPlain` function accepting a `watchNamespaces` parameter? I left that in intentionally. I assume we'll still need all of that underlying logic in order to support all install modes via the Extension API.